content/en/content-management/mathematics.md

```diff
@@ -0,0 +1,208 @@
+menu:
+  docs:
+    parent: content-management
+    weight: 250
+weight: 250
+toc: true
+math: true
+---
+
+$$
+KL(\hat{y} || y) = \sum_{c=1}^{M}\hat{y}_c \log{\frac{\hat{y}_c}{y_c}}
+JS(\hat{y} || y) = \frac{1}{2}(KL(y||\frac{y+\hat{y}}{2}) + KL(\hat{y}||\frac{y+\hat{y}}{2}))
+$$
+
+## Overview
+
+Mathematical equations and expressions authored in [LaTeX] or [TeX] are common in academic and scientific publications. Your browser typically renders this mathematical markup using an open-source JavaScript display engine such as [MathJax] or [KaTeX].
+
+For example, this is the mathematical markup for the equation displayed at the top of this page:
+
+```text
+$$
+KL(\hat{y} || y) = \sum_{c=1}^{M}\hat{y}_c \log{\frac{\hat{y}_c}{y_c}}
+JS(\hat{y} || y) = \frac{1}{2}(KL(y||\frac{y+\hat{y}}{2}) + KL(\hat{y}||\frac{y+\hat{y}}{2}))
+$$
+```
+
+Equations and expressions can be displayed inline with other text, or as standalone blocks. Block presentation is also known as "display" mode.
+
+Whether an equation or expression appears inline, or as a block, depends on the delimiters that surround the mathematical markup. Delimiters are defined in pairs, where each pair consists of an opening and closing delimiter. The opening and closing delimiter may be the same, or different. Common delimiter pairs are shown in [Step 1].
+
+The approach described below avoids reliance on platform-specific features like shortcodes or code block render hooks. Instead, it utilizes a standardized markup format for mathematical equations and expressions, compatible with the rendering engines used by GitHub, GitLab, [Microsoft VS Code], [Obsidian], [Typora], and others.
+
+## Setup
+
+Follow these instructions to include mathematical equations and expressions in your markdown using LaTeX or TeX typsetting syntax.
+
+###### Step 1
+
+Enable and configure the Goldmark [passthrough extension] in your site configuration. The passthrough extension preserves raw markdown within delimited snippets of text, including the delimiters themselves.
+
+{{< code-toggle file=hugo copy=true >}}
+[markup.goldmark.extensions.passthrough]
+enable = true
+
+[markup.goldmark.extensions.passthrough.delimiters]
+block = [['$$', '$$'], ['\[', '\]']]
+inline = [['$', '$'], ['\(', '\)']]
+
+[params]
+math = true
+{{< /code-toggle >}}
+
+The configuration above allows you to delimit inline equations and expressions using a single `$` symbol. If you need to include a single `$` to represent currency anywhere on the page, you must double-escape the `$` symbol in your markdown. For example:
+
+```text
+The price is \\$67.42, but members get a 25% discount.
+```
+
+Note that the configuration above enables mathematical rendering on every page unless you set the "math" parameter to `false` in front matter. To enable mathematical rendering as needed, set the "math" parameter to `false` in your site configuration, and set the "math" parameter to `true` in front matter.
```

Comment:
A forward reference to Step 3 might help clarify why there is a `math` parameter introduced here.

Reply:
It may also be worth noting that, even if `math` is set to false, you still should escape dollar signs if you configure them as delimiters, because otherwise the passthrough extension could prevent the rendering of other markdown syntax, as in the following example.

```
A $5 bill _saved_ is a $5 bill _earned_
```

Reply:
> otherwise the passthrough extension could prevent the rendering of other markdown syntax, as in the following example

Yeah, this is a common enough "gotcha" that the MathJax docs explain why $ isn't a default inline delimiter:
<https://docs.mathjax.org/en/latest/input/tex/delimiters.html#tex-and-latex-math-delimiters>

I think I'll do the same, removing the single $ delim from the examples, and create another section at the bottom of page describing how to enable, what to watch out for, etc.